api/src/connection/datasource.ts

```
@@ -10,27 +13,59 @@ const env = process.env;
+  // Parse the database type with proper type validation
+  const dbTypeInput = env.TR_DB_TYPE || DbType.BETTER_SQLITE3;
 
-  // Parse the port safely with fallback to 3306 if parsing fails
+  // Validate that the provided DB type is supported
+  const dbType = Object.values(DbType).includes(dbTypeInput as DbType) ? (dbTypeInput as DbType) : DbType.BETTER_SQLITE3;
+
+  // Common options for all database types
+  const commonOptions = {
+    synchronize: false,
+    logging: false,
+    entities: [__dirname + '/../entity/*.entity.{js,ts}'],
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)